docs/queries/bytestid.mdx

```diff
@@ -12,7 +12,7 @@ import TabItem from '@theme/TabItem'
 ## API
 
 ```typescript
-getByTestId(
+getByTestId({
```

Comment:
This snippet shows the TypeScript signature of the method.
The bracket is not required here.

Reply:
Oh, sorry, you are right, my bad.